integral_view/forms/disk_forms.py

```diff
@@ -0,0 +1,10 @@
+from django import forms
+
+class FormatDiskForm(forms.Form):
+    path = forms.CharField(widget=forms.HiddenInput)
+    def __init__(self, *args, **kwargs):
+      if kwargs:
+        fs_type_list = kwargs.pop('fs_types')
```

Comment:
Either check if 'fs_types' is in kwargs or give a default value as the second param or else this could throw an exception..